File: hw03-crud/src/main/java/ru/otus/shatokhin/controller/HealthController.java
package ru.otus.shatokhin.controller;

import org.springframework.http.HttpStatus;
import org.springframework.http.MediaType;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseStatus;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("${web.root}")
public class HealthController {

    @GetMapping(value = "/health", produces = MediaType.APPLICATION_JSON_VALUE)
    @ResponseStatus(HttpStatus.OK)
    public String healthCheck() {
        return "{\"status\": \"OK\"}";
    }
}

File: hw01-docker/src/main/kotlin/ru/otus/shatokhin/controller/HealthController.kt
package ru.otus.shatokhin.controller

import org.springframework.http.HttpStatus
import org.springframework.http.MediaType
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseStatus
import org.springframework.web.bind.annotation.RestController

@RestController
class HealthController {

    @GetMapping("/health", produces = [MediaType.APPLICATION_JSON_VALUE])
    @ResponseStatus(HttpStatus.OK)
    fun healthCheck(): String {
        return "{\"status\": \"OK\"}"
    }
}